generate_extra_tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# nim-eth2-scenarios
+# Copyright (c) 2018-Present Status Research & Development GmbH
+# Licensed and distributed under either of
+#   * MIT license (license terms at https://opensource.org/licenses/MIT).
+#   * Apache v2 license (license terms at https://www.apache.org/licenses/LICENSE-2.0).
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+set -eu -o pipefail
+
+VERSIONS=(
+  "v1.1.0-beta.4"
+)
+
+# signal handler (we only care about the Ctrl+C generated SIGINT)
+REL_PATH="$(dirname "${BASH_SOURCE[0]}")"
+ABS_PATH="$(cd "${REL_PATH}"; pwd)"
+cleanup() {
+	echo -e "\nCtrl+C pressed. Cleaning up."
+	cd "$ABS_PATH"
+	rm -rf consensus-specs consensus-spec-tests tests-extra-*
+	exit 1
+}
+trap cleanup SIGINT
+
+unpack_version() {
+	[[ -z "$1" ]] && { echo "usage: unpack_version() vX.Y.Z"; exit 1; }
+	version="$1"
+
+	if [[ -e "${ABS_PATH}/tests-extra-${version}" ]]; then return; fi
+
+	pushd "${ABS_PATH}"
+	rm -rf consensus-specs consensus-spec-tests tests-extra-*
+	git clone git://github.com/ethereum/consensus-specs --branch "${version}" --depth 1
```

Comment:
Where does `consensus-spec-tests` come from here? This it the only `git clone` I see, and it's for `consensus-specs`.

Also, https://github.com/status-im/nim-eth2-scenarios/blob/master/download_test_vectors.sh already gets tarballs of `consensus-spec-tests`, so that part of the machinery here seems redundant.

Apparently, the EF noticed, too, that when everyone was `git clone`ing the repo now called `consensus-spec-tests`, it wasn't ideal due to size/bandwidth, thus partially the tarballs `download_test_vectors.sh` uses instead. Perhaps `consensus-specs` is fine, though, since it's a more usual size, like the submodules already implicitly cloned by `nimbus-eth2`, rather than a gigabytes-sized respository of large binary files.

Reply:
`consensus-spec-tests` is generated by `make gen_merkle` few lines down, then renamed to `tests-extra-${version}`. This is just to start cleanly.

Reply:
`download_test_vectors.sh` downloads pre-built test vectors. It is not possible to run an additional generator on that output, it needs to be run against the spec repo.

Reply:
```
% time vendor/nim-eth2-scenarios/generate_extra_tests.sh
vendor/nim-eth2-scenarios/generate_extra_tests.sh  22.55s user 2.93s system 80% cpu 31.454 total
```